lib/clusters/cluster.rb

```diff
@@ -24,7 +24,7 @@ def inactive?
     attr_reader :uri_key
 
     def elasticsearch_config
-      @elasticsearch_config ||= ElasticsearchConfig.new.config
+      ElasticsearchConfig.new.config
```

Comment:
This involves parsing a YAML file, can we keep this cached?

Reply:
it's private and only used in the ```uri``` method which is cached

Reply:
I wonder if it would be clearer to remove the method then and have

```ruby
    def uri
      @uri ||= ElasticsearchConfig.new.config[uri_key]
    end
```

Reply:
Probably. I'll change it